CHANGELOG/CHANGELOG-1.11.md

```diff
@@ -0,0 +1,89 @@
+
+- Manage nodes in groups
+- Spread apps among node groups 
+- Run different version of app instances in different node groups
+- Limit service endpoints in the same location as the client
+
+Introduce two new APIs below to implement Node Group Management.
+
+- **NodeGroup API**: represents a group of nodes that have the same labels.
+- **EdgeApplication API**: contains the template of the application orgainzed by node groups, and the information of how to deploy different editions of the application to different node groups.
+
+Refer to the links for more details.
+([#3574](https://github.com/kubeedge/kubeedge/pull/3574), [#3719](https://github.com/kubeedge/kubeedge/pull/3719))
+
+
+### Mapper SDK
+
+Mapper-sdk is a basic framework written in go. Based on this framework, developers can more easily implement a new mapper.
+Mapper-sdk has realized the connection to kubeedge, provides data conversion, and manages the basic properties and status of devices, etc. 
+Basic capabilities and abstract definition of the driver interface. Developers only need to implement the 
+customized protocol driver interface of the corresponding device to realize the function of mapper.
+
+Refer to the links for more details.
+([#70](https://github.com/kubeedge/mappers-go/pull/70))
+
+
+
+### Beta sub-commands in Keadm to GA
+
+Some new sub-commands in keadm move to GA, including containerized deployment, offline installation, etc.
+Original `init` and `join` are replaced by `beta init` and `beta join`.
+
+- `init`: CloudCore Helm Chart is integrated in init, which can be used to deploy containerized CloudCore.
+
+- `join`: Installing edgecore as system service from docker image, no need to download from github release.
+
+- `reset`: Reset the node, clean up the resources installed on the node by `init` or `join`. It will automatically detect the type of node to clean up.
+
+- `manifest generate`: Generate all the manifests to deploy the cloudside components.
+
+
+Refer to the links for more details.
+([#3900](https://github.com/kubeedge/kubeedge/pull/3900))
+
+### Deprecation of original `init` and `join`
+
+Original `init` and `join` are deprecated, they have problems with offline installation, etc.
+
+Refer to the links for more details.
+([#3900](https://github.com/kubeedge/kubeedge/pull/3900))
+
+### Next-gen Edged to Beta: Suitable for more scenarios
+
+New version of the lightweight engine Edged, optimized from kubelet and integrated in edgecore, move to Beta.
+New Edged will still communicate with the cloud through the reliable transmission tunnel.
+
+Refer to the links for more details.
+(Dev-Branch for beta: [feature-new-edged](https://github.com/kubeedge/kubeedge/tree/feature-new-edged))
+
+
+## Important Steps before Upgrading
+
+If you want to use keadm to deploy the KubeEdge v1.11.0, please note that the keadm `init` and `join` have been replaced by `beta init` and `beta join`.
```

Comment:
If you want to use keadm to deploy the KubeEdge v1.11.0, please note that the keadm `beta init` and `beta join` have been replaced by `init` and `join`.